monitoring/mongodb/dashboard.py

```diff
@@ -1062,43 +1446,51 @@ def mongodb_state_timeline(title, expr, description="", mappings=None, **kwargs)
     "Cache I/O",
     [
         Target(
-            expr=f'sum(rate(mongodb_ss_wt_cache_bytes_{operation}_cache{{job=~"$job", namespace="{NAMESPACE}"}}[$__rate_interval]))',
-            legendFormat=f"{operation}_cache",
-        )
-        for operation in ["written_from", "read_into"]
+            expr="sum(rate(" + Metrics.SS_WT_CACHE_BYTES_WRITTEN_FROM_CACHE() + "))",
+            legendFormat="written_from_cache",
+        ),
+        Target(
+            expr="sum(rate(" + Metrics.SS_WT_CACHE_BYTES_READ_INTO_CACHE() + "))",
+            legendFormat="read_into_cache",
+        ),
     ],
     unit="binBps",
-    fillOpacity=0,
     showPoints="auto",
 )
 
 evictions_app_threads = mongodb_timeseries(
     "Evictions from application threads",
     [
         Target(
-            expr=f'rate(mongodb_ss_wt_cache_modified_pages_evicted_by_application_threads{{job=~"$job", rs_state="{RS_STATE_PRIMARY}", namespace="{NAMESPACE}"}}[$__rate_interval])',
+            expr="rate("
+            + Metrics.SS_WT_CACHE_MODIFIED_PAGES_EVICTED_BY_APPLICATION_THREADS(
+                f'rs_state="{RS_STATE_PRIMARY}"'
+            )
+            + ")",
             legendFormat="{{pod}}",
         )
     ],
     legendDisplayMode="table",
     legendPlacement="right",
     unit=UNITS.OPS_PER_SEC,
-    fillOpacity=0,
     showPoints="auto",
 )
 
 wiredtiger_cache = mongodb_timeseries(
     "Cache Size",
     [
         Target(
-            expr=f'sum({metric}{{job=~"$job", namespace="{NAMESPACE}"}}) by ()',
-            legendFormat=legend,
-        )
-        for metric, legend in [
-            ("mongodb_mongod_wiredtiger_cache_max_bytes", "max"),
-            ("mongodb_mongod_wiredtiger_cache_bytes", "used"),
-            ("mongodb_ss_wt_cache_tracked_dirty_bytes_in_the_cache", "dirty"),
-        ]
+            expr="sum(" + Metrics.MONGOD_WIREDTIGER_CACHE_MAX_BYTES() + ")",
+            legendFormat="max",
+        ),
+        Target(
+            expr="sum(" + Metrics.MONGOD_WIREDTIGER_CACHE_BYTES() + ")",
+            legendFormat="used",
+        ),
+        Target(
+            expr="sum(" + Metrics.SS_WT_CACHE_TRACKED_DIRTY_BYTES_IN_THE_CACHE() + ")",
+            legendFormat="dirty",
+        ),
```

Comment:
nit: why not use generation?

```suggestion
            expr="sum(" + metric() + ")",
            legendFormat=legend,
        )
        for metric, legend in [
            (Metrics.MONGOD_WIREDTIGER_CACHE_MAX_BYTES, "max"),
            (Metrics.MONGOD_WIREDTIGER_CACHE_BYTES, "used"),
            (Metrics.SS_WT_CACHE_TRACKED_DIRTY_BYTES_IN_THE_CACHE, "dirty"),
        ] 
```